extensions/wikia/PhalanxII/classes/Phalanx.class.php

```diff
@@ -141,7 +183,12 @@ public function save() {
 			$dbw->insert( $this->db_table, $this->mapToDB(), __METHOD__ );
 			$action = 'add';
 		} else {
-			$dbw->update( $this->db_table, $this->mapToDB(), array( 'p_id' => $this->data['id'] ), __METHOD__ );
+			$dbw->update(
+				$this->db_table,
+				$this->mapToDB(),
+				[ 'p_id' => $this->data['id'] ],
+				__METHOD__
+			);
 			$action = 'edit';
 		};
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this empty statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1116)